### ISSUE TITLE
Problems in getting switches to work

### ISSUE THREAD

**Issue by aFunkyBass** [alf45tar/PedalinoMini#106]
Hello there, I started testing Pedalino Mini, but I can't seem to make it work correctly on a ESP32-WROOM-32

First of all, to make it alive I had to disable all the environments except esp32doit-devkit-v1 in PlatformIO, otherwise it loaded the ttgo-eight-something, which apparently has a different GPIO assignment. Is the environment I'm using correct for my board?

I made a minimal test wiring with just two breadboard switches configured as Momentary 1, plugged into GPIO 25 and 26. Here's a picture of my breadboard.

![IMG_20200610_190304_DRO](https://user-images.githubusercontent.com/66510206/84298719-d6a31700-ab4f-11ea-91ce-8348b1ba7bf6.jpg)

Now, if I only select "Single press" on the pedals config page, nothing happens... If I select also "Double press" I get some messaged on the virtual display, and some feedback from the terminal. But it acts hectic sending more than a MIDI messages. Here is what I get on the terminal (I pressed once pedal 1 and once pedal2):

Pedal  1   DOUBLE PRESS
CONTROL CHANGE.....Code  12......Value  63.....Channel  1
Pedal  1   SINGLE PRESS
CONTROL CHANGE.....Code  12......Value   0.....Channel  1
Pedal  2   DOUBLE PRESS
CONTROL CHANGE.....Code  13......Value  63.....Channel  1

But the results are unpredictable, they change every time... 

Furthermore, pressing the switches I notice that the software often crashes and the board is reset.

Where am I going wrong??

P.S. PlatformIO drives me nuts!! :)

**Comment by alf45tar** [comment]
@aFunkyBass did you solved? can we close the issue?

**Comment by aFunkyBass** [comment]
> @aFunkyBass did you solved? can we close the issue?

@alf45tar sorry for being so late, but I've been working on something else in the meantime and I forgot to report to this issue.

Yes, I solved the problem. It turned out that the PIN (up/left in the picture) I've used for ground it's not actually a ground, even though it's marked as GND!!!! It took me a while to figure it out!

I have other issues now, but maybe I'll open another topic, not to mix different issues.

Thank you for you support!
